andela/persephone-ah-backend
# #166816101 add logout route

#### What does this PR do?
It adds Endpoint GET: /users/logout

#### Description of Task to be completed?
> - add moment.js
> - add /users/logout endpoint
> - add node-cron for deleting expired token
> - add blacklist table to keep track of used token

#### How should this be manually tested?
> - Clone this branch to your local machine
> - cd into the root directory
> - check the .env.sample file to set up your env file
> - run npm install and npm start
> - open your postman and create a new account
> - Then use the token generated to make a Get request to this:  http://localhost:3000/api/v1/users/logout 

#### Any background context you want to provide?
NA

#### What are the relevant pivotal tracker stories?
The pivotal [#166816101](https://www.pivotaltracker.com/story/show/166816101)

#### Screenshots (if appropriate)
NA

#### Questions:
NA
